### PR TITLE
[Images] Upload an image via a Worker

### PR DESCRIPTION
**Pull request by dcpena**
Fixed an incorrect code example and added a new page for uploading an image with a worker to address PCX-13703.

### CODE REVIEW

File: src/content/docs/images/upload-images/upload-file-worker.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+You can use a Worker to upload your image to Cloudflare Images.
+
+Refer to the example below or refer to the [Workers documentation](/workers/) for more information.
+
+```ts
+const image = await fetch("./myimage.png");
+const bytes = await image.bytes();
+formData.append('file', new File([bytes], 'myimage.png');
+```
```
Comment (dochne) [inline src/content/docs/images/upload-images/upload-file-worker.mdx]: We should extend this so it's a bit less of a stub!  This guides them through the weird bit, but an example that they can copy/paste the majority of is always preferable!

```ts
const API_URL = "https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/images/v1";
const TOKEN = "YOUR_TOKEN_HERE";

const image = await fetch("https://example.com/image.png");
const bytes = await image.bytes();

const formData = new FormData();
formData.append('file', new File([bytes], 'image.png'));

const response = await fetch(API_URL, {
  method: 'POST',
  headers: {
    'Authorization': `Bearer ${TOKEN}`,
  },
  body: formData,
});
```

**Comment by dochne** [comment]
I think it's worth us adding an example here phrased as "Uploading from AI Generated Images" with this code in it:
```ts
const API_URL = "https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/images/v1";
const TOKEN = "YOUR_TOKEN_HERE";

const stream = await env.AI.run(
  "@cf/bytedance/stable-diffusion-xl-lightning",
  {
    prompt: YOUR_PROMPT_HERE
  }
);
const bytes = await (new Response(stream)).bytes();

const formData = new FormData();
formData.append('file', new File([bytes], 'image.jpg');

const response = await fetch(API_URL, {
  method: 'POST',
  headers: {
    'Authorization': `Bearer ${TOKEN}`,
  },
  body: formData,
});
```

File: src/content/docs/images/upload-images/upload-file-worker.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+pcx_content_type: how-to
+title: Upload an image using a Worker
```
Comment (dochne) [inline src/content/docs/images/upload-images/upload-file-worker.mdx]: image

Looking at the titles of our current options, "Upload via Worker" seems like it'd fit better

File: src/content/docs/images/upload-images/upload-file-worker.mdx (hunk 1 above)
Comment (dcpena) [reply src/content/docs/images/upload-images/upload-file-worker.mdx]: Thanks so much for this! I definitely agree that an example that walks a user through _and_ is something they can copy/paste is preferable! Updated

File: src/content/docs/images/upload-images/upload-file-worker.mdx (hunk 2 above)
Comment (dcpena) [reply src/content/docs/images/upload-images/upload-file-worker.mdx]: Done

File: src/components/models/code/TextToImageCode.astro [hunk 3]
```diff
@@ -23,14 +23,14 @@ export default {
       prompt: "cyberpunk cat",
     };
 
-    const response = await env.AI.run(
+    const stream = await env.AI.run(
       "${name}",
       inputs
     );
 
     return new Response(response, {
```
Comment (pedrosousa) [inline src/components/models/code/TextToImageCode.astro:31]: Should we also change `response` here to `stream`?

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**